assets/styles/layouts/_global.scss

```diff
@@ -292,7 +292,7 @@ html {
 
 
 /* Small Devices, Tablets */
-@media only screen and (max-width: 768px) {
+@media only screen and (max-width: 1024px) {
```

Comment:
Can we eliminate using raw media queries`media only screen and (max-width: 1024px)` and use `@include media()` instead?